The Debian packaging of farbfeld is maintained using dgit.  For the sake
of an efficient workflow, Debian modifications to the upstream
source are squashed into a single diff, rather than a series of
quilt patches.  To obtain a patch queue for package version
2.20170109-1:

    # apt-get install dgit
    % dgit clone foo
    % cd foo
    % git log --oneline 2.20170109-1..debian/2.20170109 -- . ':!debian'
--- farbfeld-2.20170109.orig/config.mk
+++ farbfeld-2.20170109/config.mk
@@ -4,8 +4,8 @@ VERSION = 2
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
-MANPREFIX = ${PREFIX}/man
+PREFIX = /usr
+MANPREFIX = ${PREFIX}/share/man
 
 PNGLIB = /usr/local/lib
 PNGINC = /usr/local/include
@@ -17,9 +17,9 @@ INCS =
 LIBS =
 
 # flags
-CPPFLAGS = -D_DEFAULT_SOURCE
-CFLAGS   = -std=c99 -pedantic -Wall -Os ${INCS}
-LDFLAGS  = -s ${LIBS}
+CPPFLAGS += -D_DEFAULT_SOURCE
+CFLAGS   += -std=c99 -pedantic -Wall -Os ${INCS}
+LDFLAGS  += -s ${LIBS}
 
 # compiler and linker
 CC = cc
